src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -354,8 +354,12 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
       body shouldBe ""
       status shouldBe NO_CONTENT_204
 
+      val bagStoreBagId = new PropertiesConfiguration((depositDir / "deposit.properties").toJava)
+        .getString("bag-store.bag-id")
+      bagStoreBagId shouldNot be(null)
```

Comment:
```suggestion
      bagStoreBagId should not be null
```

Reply:
or even better, use `new PropertiesConfiguration((depositDir / "deposit.properties").toJava).containsKey("bag-store.bag-id") shouldBe true`

Reply:
Need `bagStoreBagId` on the next line

Reply:
then keep the value, but use `.containsKey`